### DISCUSSION TITLE
local snapshot with viewer

### DISCUSSION THREAD

**Discussion by 7758521momo** [iTwin/itwinjs-core#3643] [Q&A] [answered]
Can I use The @itwin/web-viewer-react or @itwin/desktop-viewer-react opening local snapshots?
Thank you!

**Comment by aruniverse** [comment, marked as answer]
Yes you can use the itwin/desktop-viewer-react to open a local snapshot. As mentioned previously you can create a new desktop app like the web app.

**Comment by pmconne** [comment]
@7758521momo this is an open discussion forum where others can search for and benefit from answers to previously-asked questions.
Please refrain from editing your answered questions to remove their content.
